AltairZ80/s100_2sio.c

```diff
@@ -769,15 +790,16 @@ static int32 m2sio_stat(DEVICE *dptr, int32 io, int32 data)
         if ((data & M2SIO_RESET) == M2SIO_RESET) {
             sim_debug(STATUS_MSG, dptr, "MC6850 master reset.\n");
             xptr->stb &= (M2SIO_CTS | M2SIO_DCD);           /* Reset status register */
-            xptr->rxb = 0;
-            xptr->txp = 0;
-            xptr->tie = 1;
-            xptr->rie = 1;
+            xptr->rxb = 0x00;
+            xptr->txp = FALSE;
+            xptr->tie = FALSE;

```

Comment:
Please double check the new initialization for `tie` and `rie` is now `FALSE` (was 1 = `TRUE`).

Reply:
Interrupts are supposed to be disabled by default.